vscode-extension/editor/src/VSCodeEditor.tsx

```diff
@@ -124,6 +138,8 @@ export default function VSCodeEditor() {
     const res = await ufetch.post(route, {});
     // console.log(`IN LOAD: route=${route}, res=${JSON.stringify(res)}`);
     setAIConfig(res.aiconfig);
+    // If we can load the config from the server, we assume it's in a good state
+    setIsReadOnly(false);
```

Comment:
Should we also update the webview state in this case?
i.e. `updateWebviewState(vscode, { isReadOnly: false });`

Reply:
Ah, good catch - yep!

Reply:
updated